tools/data_manipulation/xarray/test-data/version.tabular

```diff
@@ -0,0 +1 @@
+Galaxy xarray version 0.18.2
```

Comment:
one o those file is not needed?

Reply:
Why? Should I always have `version.tabular`?

Reply:
Not always but you have two tools that generate two different files with the same content, is that like it should be?

btw. do you know: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-version-command